src/views/pages/_package.html

```diff
@@ -14,6 +14,30 @@
 	description="{{description}}"
 	openGraphImage="{{#if type === 'npm'}}{{@shared.serverHost}}/open-graph/image/npm/{{name}}{{/if}}"
 >
+	{{#partial seo}}
+		<script type="application/ld+json">
+			{
+			"@context": "https://schema.org",
+			"@type": "WebPage",
+			"mainEntity": {
+				"@context":"http://schema.org",
+				"@type":"SoftwareSourceCode",
+				"name":"{{name}}",
+				{{#if versionP}}"version":"{{versionP}}",{{/if}}
```

Comment:
You can define all this in a real JS as component data and then just do `{{{JSON.stringify(...)}}}` here

Reply:
Sure thing. I moved it `computed` although I'm not 100% sure if that's the right area to keep it in. Feel free to let me know if I there's a better way of doing it :+1: